packages/tests/helpers/index.ts

```diff
@@ -41,10 +46,10 @@ export const setup = async (
     let dbConfig: Knex.Config = {
         client: 'mysql',
         connection: {
-            host: '127.0.0.1',
-            user: 'root',
-            password: '',
-            database: 'testdb'
+            host: process.env.MYSQL_HOST || '127.0.0.1',
```

Comment:
Please use `DATABASE_HOST`, `DATABASE_USER` etc instead. Its more reusable that way. Far less variables to remember.

Reply:
because we have different databases and would have more when we put `DATABASE_HOST` which would we be referring to?

Reply:
it will refer to whatever database is being used. If we support 12 databases, they all will have a database host. So instead of someone remembering MYSQL_HOST, PG_HOST, DYNAMO_HOST etc etc, they just provide DATABASE_HOST and whatever database uses that env.